web/templates/layout/header.html.eex

```diff
@@ -5,31 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="A package manager for the Erlang ecosystem">
-    <link rel="search" type="application/opensearchdescription+xml" title="Hex" href="/hexsearch.xml">
-    <link rel="stylesheet" href="<%= static_path(HexWeb.Endpoint, "/css/app.css") %>">
+
     <title><%= title(assigns) %></title>
+
+    <link rel="search" type="application/opensearchdescription+xml" title="Hex" href="/hexsearch.xml">
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,300,700,900,100' rel='stylesheet' type='text/css'>
```

Comment:
Yes, I can add in main.scss but not clear if it's better if we use like that, because it's using a CDN and will definitely load faster if we use google version. Haven't tested also if it will break with sass. I'll play with that today.